general:
  system: 1     # 0: IMOMD-RRT;  1: Bidirectional A*;  2: ANA*
  pseudo: 0     # 0: real-objectives mode;  1: pseudo-objectives mode
  log_data: 1   # 0: don't log;             1: log data
  print_path: 0 # 0: don't print;           1: print path (sequence of node_id)
  max_iter: 1000000
  max_time: 60 # Sec

rrt_params:
  goal_bias: 1.0 # 0.0 ~ 1.0
  random_seed: 0 # 0: Fixed random_seed; 1: use random random_seed

destinations:
  ## sanfrancisco_bugtrap (Pseudo-mode)
  # source_id: 808563
  # objective_ids: [566163, 394039, 1183351, 835647, 725115, 393810,  1241804, 1053685]
  # target_id: 560879

  ## Seattle
  source_id: 486234
  objective_ids: [433402, 426716, 617432, 887981, 469267,
                  660438, 271555, 209507, 814027, 538265,
                  112753, 398120, 623085, 782523, 289206,
                  377816, 367108, 401203, 725126, 870459,
                  323008, 525033, 594246]
  target_id: 683162

map:
  type: 1 # < 0: customized map number; 0: customized graph, 1: osm
  path: ../../../src/IMOMD-RRT/osm_data/
  name: Seattle.osm # seattle.osm

rtsp_settings:
  shortcut: 1  # 0: No Execute,  1: Execute
  swapping: 1  # 0: No Execute,  1: Execute
  genetic:  1  # 0: No Execute,  1: Execute
  ga:
    random_seed: 0 # 0: Fixed random_seed; 1: use random random_seed
    mutation_iter: 10000
    population: 1000
    generation: 5
